subprojects/gradle/lint-report/src/integTest/resources/lint-results.html

```diff
@@ -0,0 +1,1534 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
You can try `<!-- //NOSONAR -->` to suppress sonarqube

https://github.com/SonarSource/sonar-html/blob/cda2f6bff96d9dc459d92db32199a91fb9a8e1b3/sonar-html-plugin/src/test/java/org/sonar/plugins/html/visitor/NoSonarScannerTest.java#L45